canvas_modules/common-canvas/__tests__/common-canvas/common-canvas-test.js

```diff
@@ -31,54 +33,82 @@ describe("CommonCanvas renders correctly", () => {
 		canvasController = new CanvasController();
 	});
 
+	it("should render one <CanvasBottomPanel/> component when showBottomPanel is true", () => {
+		const config = {};
+		const wrapper = createCommonCanvas(config, canvasController, { showBottomPanel: true }, {});
+		expect(wrapper.find(CanvasBottomPanel)).to.have.length(1);
+		expect(canvasController.isBottomPanelOpen() === true).to.be.true;
+	});
+
+	it("should render one <CanvasBottomPanel/> component when showBottomPanel is false", () => {
+		const config = {};
+		const wrapper = createCommonCanvas(config, canvasController, { showBottomPanel: false }, {});
+		expect(wrapper.find(CanvasBottomPanel)).to.have.length(1);
+		expect(canvasController.isBottomPanelOpen() === false).to.be.true;
+	});
+
+	it("should render one <CommonCanvasRightFlyout/> component when showRightFlyout is true", () => {
+		const config = {};
+		const wrapper = createCommonCanvas(config, canvasController, {}, { showRightFlyout: false });
+		expect(wrapper.find(CommonCanvasRightFlyout)).to.have.length(1);
+		expect(canvasController.isRightFlyoutOpen() === true).to.be.false;
```

Comment:
Something strange is going on here. You set the `showRightFlyout` to `false` but you are testing for `true`.